_posts/en/newsletters/2021-06-02-newsletter.md

```diff
@@ -0,0 +1,124 @@
+  alternative transaction selection algorithm for miners.  Bitcoin's
+  consensus rules enforce that no transaction can be included in a block
+  unless all of its unconfirmed ancestors are also included earlier in
+  that same block.  Bitcoin Core addresses this constraint by treating
+  each transaction with unconfirmed ancestors as if it contained both
+  the fees and the size of those ancestors.  For example, transaction B
+  depends on unconfirmed transaction A, so Bitcoin Core adds together
+  the fees paid by both transactions and divides them by the combined
+  size of both transactions.  This allows Bitcoin Core to fairly compare
+  all transactions in the mempool based on their effective feerate
+  whether or not those transactions have any ancestors.
+
+    However, Erhardt and Shikhelman note that a more sophisticated
+    algorithm that may require a bit more CPU can find sets of related
+    transactions that are even more profitable to mine than Bitcoin
+    Core's existing simple algorithm.  The authors tested their
+    algorithm on historic mempool data and found that it would've
+    collected slightly more fees than Bitcoin Core's existing algorithm
+    in almost all recent blocks.
+
+    If implemented and used by miners, the improved algorithm could
+    allow multiple users who each received an output from a large
+    [coinjoin][topic coinjoin] or [batched payment][topic payment
+    batching] to each pay a small part of the total fee necessary [CPFP
+    fee bump][topic cpfp] that coinjoin or payment.  That would be an
+    improvement over the current case where each user's CPFP fee bump is
+    considered independently and multiple related fee bumps may not have
+    an aggregate effect on whether an ancestor transaction gets mined.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- FIXME:harding hwi
+
+- [LND 0.13.0-beta.rc3][LND 0.13.0-beta] is a release candidate that
+  adds support for using a pruned Bitcoin full node, allows receiving
+  and sending payments using Atomic MultiPath ([AMP][topic multipath payments]),
+  and increases its [PSBT][topic psbt] capabilities, among other improvements
+  and bug fixes.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #20833][] is the first PR in an effort to bring [package
+  relay][topic package relay] to Bitcoin Core. This change allows the
```

Comment:
I think it'd be better to emphasize that this PR is for mempool package acceptance. Future work may build package relay on top of this work, but best not to overpromise right now.

Reply:
"Bitcoin Core #20833 is the first PR in an effort to implement mempool package acceptance in Bitcoin Core. This change allows the `testmempoolaccept` RPC to accept multiple transactions. Future PRs will enable testing L2 transaction chains, submitting transaction packages directly to the mempool through RPCs and communicating packages over the P2P network."

Reply:
(emphasis on may)

Reply:
Pushed a commit taking these suggestions and adding a contextual link for mempool package acceptance.